README.md

```diff
@@ -305,12 +305,12 @@ the property name of the source Object, no `DataFetcher` is needed.
 Example of configuring a custom `DataFetcher`:
 ```java
 
-DataFetcher calculateComplicatedValue = new DataFetcher() {
+DataFetcher calculateComplicatedValue = new DataFetcher<ComplicatedVal>() {
```

Comment:
`DataFetcher<ComplicatedVal>` on the left; but maybe instead of `ComplicatedVal` something simpler like `Foo` or `Bar`?